frontend/src/metabase/query_builder/components/filters/modals/InlineOperatorSelector/InlineOperatorSelector.styled.ts

```diff
@@ -0,0 +1,36 @@
+import styled from "@emotion/styled";
+import { space } from "metabase/styled-components/theme";
+import { color, alpha, darken, lighten } from "metabase/lib/colors";
+
+export const InlineOperatorContainer = styled.div`
+  font-weight: bold;
+  margin: ${space(1)} 0;
+`;
+
+export const FieldTitle = styled.span`
+  color: ${color("text-dark")};
+`;
+
+export const OperatorDisplay = styled.button`
+  font-weight: bold;
+  color: ${props => (props.onClick ? color("brand") : color("text-light"))};
+  ${props => (props.onClick ? "cursor: pointer;" : "")} :hover {
```

Comment:
Maybe adding a new line at `:hover` would make this easier to read. Also, just noticed other places use `&:hover` rather than just purely `:hover` I guess it works both ways, but `&:hover` seems more established from a quick search.